src/App.js

```diff
@@ -41,6 +42,19 @@ const App = () => {
       .then((user) => user && setIsOrgAdmin(user.identity.user.is_org_admin));
   }, []);
 
+  axiosInstance.interceptors.request.use(async (config) => {
```

Comment:
Can we add this to frontend-components? This way it will be applied to every other app and we don't have to do it manually.

Reply:
@karelhala  Ideally yes, but it increases the size of the header. And that might be an issue in a near-future :/

Reply:
Let's slip this trough, but we should bite the buller soon-ish.